NetKAN/PolishFlagPack.netkan

```diff
@@ -6,7 +6,7 @@
     "author"       : [ "Venthe" ],
     "abstract"     : "Small pack of Polish flags",
     "license"      : "CC-BY-SA-4.0",
-    "ksp_version"  : "1.1.3",
+    "ksp_version"  : "1.2.2",

```

Comment:
If this is nothing but flags, you could change that to 

    "ksp_version" : "any",
    "comment"     : "flagmod",

And then you won't need to update it again.